front/app/containers/Admin/invitations/index.tsx

```diff
@@ -18,12 +19,12 @@ const InvitationsPage = () => {
   const tabs = [
     {
       label: formatMessage(messages.tabInviteUsers),
-      url: '/admin/users/invitations',
+      url: '/admin/users/invitations' as RouteType,
       name: 'index',
     },
     {
       label: formatMessage(messages.tabAllInvitations),
-      url: '/admin/users/invitations/all',
+      url: '/admin/users/invitations/all' as RouteType,
       name: 'all',
```

Comment:
@EdwinKato Seems a bit odd that we need to do casting here - should we make a "TabType" or something instead do you think? Or is this casting ok? :)